domains/rajat.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "rajat-malhotra0",
+    "email": "rajat15id@gmail.com"
+  },
+  "records": {
+    "CNAME": "https://rajat-malhotra0.github.io/portfolio/"
```

CNAME doesnt allow any trailing slashes (/)
```suggestion
    "CNAME": "rajat-malhotra0.github.io"
```